docs/ops-recipe/harborDownOrUnhealthy.md

```diff
@@ -0,0 +1,313 @@
+---
+
+
+title: "Harbor Down/Harbor Unhealthy"
+
+  
+---
+
+This alert fires because a component (eg core, registry, portal etc) of Harbor is either down or unhealthy.
```

Comment:
It might worth a single sentence what each components does / is responsible for. For example what does registry do? What is stored in the database? How crucial is is to loose the data on it? Portal is merely the UI?

I think it help evaluating the situation for someone getting paged for it who is not necessarily that familiar with Harbor in the Platform Area.